docs/architecture/data-sources.md

```diff
@@ -32,14 +32,14 @@ dotnet add package Microsoft.Extensions.DependencyInjection
 ### Adding/Extending a Data Source Interface
 Each new data source should extend from a generic data source interface. A data source interface defines all the parameters and functions that any data source that falls under it's purview must define/implement. By defining the interface, it allows the SDK to switch between the set of data sources seamlessly because they all share the same input functions and output types. 
 
-Currently there is one data source interface defined -  `ICarbonIntensityDataSource` - which is the interface for all data sources that provide carbon intensity data. A new data source interface should be defined only when there is a new general area of calculation that is being introduced to the SDK.
```

Comment:
the second part of the sentence doesn't quite make sense anymore. Maybe something along the lines of " - which cover data sources that provide carbon intensity data" or something like that